samples/CartCreate/SimpleCartCreate.php

```diff
@@ -24,13 +24,16 @@
 use ApaiIO\Operations\CartCreate;
 
 $conf = new GenericConfiguration();
+$client = new \GuzzleHttp\Client();
+$request = new \ApaiIO\Request\Rest\Request($client);
```

Comment:
I would declare the name of the request class into more specific one. The request is some kind a Adaptor for your library and therefore I would choose something like GuzzleRequest or similar.